pydra/engine/state.py

```diff
@@ -193,7 +205,9 @@ def left_splitter_rpn(self):
 
     @property
     def left_splitter_rpn_compact(self):
-        # left rpn part, but keeping the names of the nodes, e.g. [_NA, _NB, *]
+        """ the Left Part of the splitter using RPN in a compact form,
+        (without unwrapping the states from previous nodes), e.g. [_NA, _NB, *]
```

Comment:
```suggestion
        (without unwrapping the states from previous nodes), e.g. [\_NA, \_NB, \*]
```

these are reserved characters in both restructured text and markdown.